app-rating/src/main/java/com/stepstone/apprating/AppRatingDialogView.kt

```diff
@@ -0,0 +1,175 @@
+/*
+Copyright 2017 StepStone Services
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.stepstone.apprating
+
+import android.content.Context
+import android.content.res.Resources
+import android.os.Build
+import android.support.annotation.ColorRes
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.EditText
+import android.widget.LinearLayout
+import android.widget.TextView
+
+import com.stepstone.apprating.listener.OnRatingBarChangedListener
+import com.stepstone.apprating.ratingbar.CustomRatingBar
+
+/**
+ * This class represents custom dialog view which contains
+ * rating bar, edit box and labels.
+ */
+class AppRatingDialogView(context: Context) : LinearLayout(context), OnRatingBarChangedListener {
+
+    private var ratingBar: CustomRatingBar? = null
```

Comment:
Maybe you could replace `private var ratingBar: CustomRatingBar? = null` with `private lateinit var ratingBar: CustomRatingBar` and others similarly?
https://kotlinlang.org/docs/reference/properties.html#late-initialized-properties